foqus_lib/gui/tests/test_ml_ai.py

```diff
@@ -196,6 +202,7 @@ def test_flowsheet_run_successful(
 
     def test_load_and_run_meacustomnormformjson(self, active_session, simnode):
         pytest.importorskip("tensorflow", reason="tensorflow not installed")
+        pytest.importorskip("json", reason="json not installed")
```

Comment:
Similarly to `pickle`, we don't need to handle the case where `json` is not importable.